src/containers/flows/BuyNow/BuyNow.tsx

```diff
@@ -73,6 +74,17 @@ const BuyNow = ({
 		);
 	}
 
+	if (step === Step.FailedToCheckZAuction) {
+		return WrapWizard(
+			<Wizard.Confirmation
+				error={error}
+				message={'Failed to check zAuction approval status'}
+				primaryButtonText={'Close'}
+				onClickPrimaryButton={onCancel}
+			/>,
+		);
+	}
```

Comment:
Okay I updated this to include an extra ```setCurrentStep``` when failing zAuction approval. From memory, this will need to be updated on other flows as well. I think having the 'close' button and failed zAuction approval step were initially by design but it does make sense to include a retry option, rather than close the modal and re-open the flow. The update I've made is shown below. Let me know what you think, works for me, could be one for design though..

https://user-images.githubusercontent.com/39112648/177580147-7a05b3b3-6788-4ac7-8404-5c192b6faec1.mov